package.json

```diff
@@ -73,6 +73,8 @@
     "watch": "watch 'npm test --silent' src --interval 1",
     "test": "FORCE_COLOR=1 TAP_COLORS=1 npm-run-all build:clean -p build:mjs build:js -s build:prettier -p lint:* tap:* -c --aggregate-output --silent",
     "prepublishOnly": "npm test",
+    "test:js": "FORCE_COLOR=1 TAP_COLORS=1 npm-run-all build:clean build:js tap:js",
+    "test:mjs": "FORCE_COLOR=1 TAP_COLORS=1 npm-run-all build:clean build:mjs tap:mjs",
```

Comment:
I'm not following the reasons for this change, but I have a new approach to scripts that I'll copy over soon from graphql-api-koa and graphql-react: https://github.com/jaydenseric/graphql-api-koa/blob/master/package.json#L58. It leverages `prepare` so people can npm install a git url to use forks and even specific commits.